### PR TITLE
Update to Vue 3.2 

### PR DESCRIPTION
**Pull request by HoloPollock**
update to vue 3.2

### CODE REVIEW

**Review by qjack001 — APPROVED** [review]

**Review by qjack001 — CHANGES_REQUESTED** [review]
wait — where did the controls UI go?

File: package.json [hunk 1]
```diff
@@ -11,11 +11,11 @@
 		"core-js": "^3.6.5",
 		"glob-parent": "5.1.2",
 		"vite": "^2.3.8",
-		"vue": "^3.0.0",
+		"vue": "^3.2.2",
 		"vue-tippy": "^6.0.0-alpha.30"
 	},
 	"devDependencies": {
-		"@vue/compiler-sfc": "^3.0.0"
+		"@vue/compiler-sfc": "^3.2.2"
```
Comment (qjack001) [inline package.json:18]: Is this still nessesarry? It's not just included with the move to vue 3.2?

Comment (HoloPollock) [reply package.json:18]: No clue lol

**Comment by HoloPollock** [comment]
Oh yah weird

File: src/components/Controls.vue [hunk 2]
```diff
@@ -38,7 +38,7 @@
 	
 	const currentYear = new Date().getFullYear()
 	const selectedYear = ref(currentYear)
-	const emit = defineEmit(["changeUsername", "changeYear"])
+	const emit = defineEmits(["changeUsername", "changeYear"])
```
Comment (HoloPollock) [inline src/components/Controls.vue:41]: seemed like this changed

**Comment by HoloPollock** [comment]
also I added package lock

**Review by qjack001 — APPROVED** [review]

**Pull request merged**